src/algebra/hom/equiv.lean

```diff
@@ -94,16 +94,13 @@ variables (F)
 @[priority 100, -- See note [lower instance priority]
   to_additive]
 instance [has_mul M] [has_mul N] [h : mul_equiv_class F M N] : mul_hom_class F M N :=
-{ coe := (coe : F → M → N),
-  coe_injective' := @fun_like.coe_injective F _ _ _,
-  .. h }
+{ .. h }
```

Comment:
We should actually repeat the data fields when using `old_structure_cmd`, the reasoning is a bit complicated but the main thing is it ensures better unfolding.
```suggestion
{ coe := (coe : F → M → N),
  .. h }
```

Reply:
It should probably actually look like (untested):
```suggestion
{ coe := coe_fn,
  .. h }
```